secret_words.py

```diff
@@ -0,0 +1,17 @@
+"""Interface for reading and writing the secret words file in javascript format"""
+
+SECRET_WORDS_FILE = "static/assets/js/secretWords.js"
+
+
+def read():
```

Comment:
why not pass the location of the file as a parameter?

also this may be out of scope of this PR, but why js is the format of the secret words and not JSON?